cmd/root_test.go

```diff
@@ -229,3 +229,70 @@ func TestArgsParsing_CreateEmptyFileFlag(t *testing.T) {
 		})
 	}
 }
+
+func TestArgParsing_ExperimentalMetadataPrefetchFlag(t *testing.T) {
```

Comment:
Let's [write a different test function for error scenario.](https://g3doc.corp.google.com/company/teams/go-community/gotip/episodes/50.md?cl=head).

Reply:
Done!